integrations.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: page
+title: Integrations
+permalink: /integrations/
+redirect_from:
+  - /source-code/
+  - /downloads/
+---
+
+
+<a name="packages"></a>
+### Packages/Samples
+- <a href="https://github.com/rbtech/fuse-lottie" target="_blank"> Lottie animations: integration of Lottie animations on Fuse</a>
+- <a href="https://github.com/naosk8/fuse-homography-image-uploader" target="_blank"> Homography image transformation: A sample project for Fuse, an image uploader supporting Homography image transformation.</a>
+- <a href="https://github.com/jeroensmeets/mastodon-app" target="_blank"> 11t: 11t is an iOS and Android app for connecting to Mastodon</a>
```

Comment:
I think this would be better as either a data-file (see the _data folder), or simply as a few lists in the front-matter that gets looped over...

Reply:
By the way, I think just adding it as frontmatter is both the simplest and cleanest solution.

Reply:
Done this here:
https://github.com/kusma/fuse-open.github.io/tree/integrations-page

But thinking about it a little bit more, perhaps it would be even cleaner if this was done using collections? That way we could use markdown in the description as well, and avoid putting HTML in the yml... Oh well, that can be done later, I suppose.